### PR TITLE
md spec for oog log error

### PR DESCRIPTION
**Pull request by DreamWuGit**
- markdown spec for oog log error
- TODO: add python code will add after circuit https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1116 merges.

### CODE REVIEW

File: specs/error_state/ErrorOutOfGasLog.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+1. stack reads `mstart`, `msize` for gas calculation
+2. `gas_left < gas_cost`.
+3. error common constraint:
+  - current call must be failed.
+  - rw_counter_end_of_reversion constraint
+  - If it's a root call, it transits to `EndTx`. 
+  - if it is not root call, it restores caller's context by reading to `rw_table`, then does step state transition to it.
+
+## Code
+    TODO: add python code after circuit merges!
```
Comment (CPerezz) [inline specs/error_state/ErrorOutOfGasLog.md:24]: Can we file an issue for that??

Comment (DreamWuGit) [reply specs/error_state/ErrorOutOfGasLog.md:24]: #374 target this.

**Comment by aguzmant103** [comment]
@DreamWuGit @adria0 @CPerezz is this ready to be merged?

**Review by KimiWu123 — APPROVED** [review]
LGTM!

**Pull request merged**